lightning/src/ln/chan_utils.rs

```diff
@@ -335,7 +350,7 @@ pub struct ChannelPublicKeys {
 	/// states.
 	pub revocation_basepoint: PublicKey,
 	/// The public key which receives our immediately spendable primary channel balance in
-	/// remote-broadcasted commitment transactions. This key is static across every commitment
```

Comment:
Not your bug, but can you fix this comment to not refer to ours/theirs/counterparty/etc - this struct is generic across both sides (and thus should only ever use broadcaster/countersignatory).

Reply:
Hmmm I keep the counterparty's it makese sense here, as this immediate spend is only available if it's the counterparty commitment ?